### PR TITLE
2.2 bake templates improvements

### PR DESCRIPTION
**Pull request by rchavik**
Rebase of @ionas' PR: #628, #629, #630, #631, #633 

Please review and comment. CC: @croogo/contributors

### CODE REVIEW

**Comment by jhli** [comment]
Looks good to me.

Nitpicky: "Invalid foo." or "No." (in some cirumstances), are complete sentencens. Any reason aside BC to not have fullstops?

And: i do often omit the last ; when using <?= ?> or <?php echo ?> or <?php foreach(): ?> <?php endforeach ?>

I noticed the style is mixed now (even in your PR/commit diff)

**Comment by rchavik** [comment]
Yes, the missing full stops is because BC reasons.  We have a bunch of mismatch now that needs to be cleaned up. :(

Re: About the missing `;`, I think that was from the original format in the PR.  I always used the closing `;`.

**Comment by rchavik** [comment]
Noticed some weird formatting for row actions in admin_index, eg: each action is given its own column. Will need to fix this before merging

**Comment by jhli** [comment]
Thanks fir the fixed's!

File: Croogo/Console/Templates/croogo/views/admin_view.ctp [hunk 1]
```diff
@@ -68,7 +68,16 @@ foreach ($fields as $field) {
...
+				break;
+			case 'boolean':
+				echo "\t\t<dd>\n\t\t\t<?php echo \$this->Html->status(\${$singularVar}['{$modelClass}']['{$field}']); ?>\n\t\t\t&nbsp;\n\t\t</dd>\n";
+				break;
+			default:
+				echo "\t\t<dd>\n\t\t\t<?php echo h(\${$singularVar}['{$modelClass}']['{$field}']); ?>\n\t\t\t&nbsp;\n\t\t</dd>\n";
+		}
 	}
 }
 ?>
```
Comment (ionas) [inline Croogo/Console/Templates/croogo/views/admin_view.ctp]: This is the only change for admin_view but there should be more. I am not utilizing admin_view templates at the moment and thus I did not notice. Generated content looks like this
```php
///...
		<dt><?php echo __d('croogo', 'Name'); ?></dt>
		<dd>
			<?php echo h($tariff['Tariff']['name']); ?>
			&nbsp;
		</dd>
//...
```
E.g. the translation domain is still wrong for those files.

**Comment by ionas** [comment]
I will try to prepare another patch to fix that this week. Sorry for the slowdown but I have been very busy and headachey with denormalizing data via sql.

**Pull request merged**